tpgtools/README.md

```diff
@@ -62,57 +62,57 @@ Every resource added via tpgtools needs an override file for every version it is
 Override files contain information on how the Terraform representation of the resource should be different from the DCL's representation. This could be naming or behavior differences, but for a new resource implemented through tpgtools there should be no differences from the DCL's representation.
 
 ### Adding Samples
+For a deeper understanding on test anatomy please read the accompanying [Tests and Sample Anatomy](#tests-and-sample-anatomy)
 
-Adding samples is essential for generating tests and documentation.
+#### Create a meta.yaml file
+Create a meta.yaml file in the overrides directory for the resource. For example,
+[this meta.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/tpgtools/overrides/assuredworkloads/samples/workload/meta.yaml)
+file exists for the assured workloads resource. This file will merge with any tests. You can customize behavior
+of the tests and examples generated dcl samples data (injections, hiding, ect..). See [the section of the meta.yaml file](#the-metayaml-file)
+for a more detailed dive.
 
+#### Adding DCL Tests
 Start by copying the relevant samples from the DCL for your new resource. These will be added to
 the [tpgtools/api folder](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/tpgtools/api) under the relevant product.
 These samples can be found under the samples/ folder within the DCL for the resource being added. For example, assured
 workloads can be found [here](https://github.com/GoogleCloudPlatform/declarative-resource-client-library/tree/main/services/google/assuredworkloads/samples).
 
 Re-serialize `serialization.go` to enable generating configs from samples by running:
 ```
-make upgrade-dcl
 make serialize
 ```
 
-Create a meta.yaml file in the overrides directory for the resource. For example, 
-[this meta.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/tpgtools/overrides/assuredworkloads/samples/workload/meta.yaml) file exists for the assured workloads resource.
-This file will merge with any tests you customize behavior
-of the tests and examples generated from the samples data (injections, hiding, ect..)
+#### Adding a Non DCL Test
+If you need to write tests manually you can add terraform templates to
```

Comment:
Maybe add one sentence explaining why some tests need to be implemented manually?

Reply:
done